src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1460,7 +1438,7 @@ void GLHEVert::makeThisGLHECacheStruct()
 
     auto &d_bh_data = d["BH Data"];
     int i = 0;
-    for (auto &thisBH : this->myRespFactors->myBorholes) {
+    for (auto const &thisBH : this->myRespFactors->myBorholes) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:1463]:(style),[constVariable],Variable 'thisBH' can be declared as reference to const